app/controllers/env_settings_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class EnvSettingsController < ApplicationController
+
+  def index
+    render json: values
+  end
+
+  private
+
+  def values
+    {
+      sentry: ENV['SENTRY_DSN_FRONTEND'],
+      rails_port: ENV['RAILS_PORT'],
```

Comment:
I think we don't need this one. we can just get the one while building the image.